cmd/limactl/protect.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/lima-vm/lima/pkg/store"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+func newProtectCommand() *cobra.Command {
+	var protectCommand = &cobra.Command{
+		Use:   "protect INSTANCE [INSTANCE, ...]",
+		Short: "Protect an instance to prohibit accidental removal",
+		Long: `Protect an instance to prohibit accidental removal via the 'limactl delete' command.
+An instance can be still removable via '/bin/rm', Finder, etc.`,
```

Comment:
Sounds weird to me; I would say "can still be removed" or "is still removable".

Or maybe even: "The instance is not being protected against removal via ...".

Reply:
Fixed